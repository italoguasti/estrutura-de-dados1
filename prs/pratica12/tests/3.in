16 manjericao fazendatrescachoeiras
1 arroz fazendadoamanhecer
91 brocolis sitioamanhecer
180 coentro fazendariogrande
143 alface sitiohorizonteazul
198 maracuja fazendacamposereno
164 batata sitioserraazul
184 brocolis fazendabelacolina
90 tabaco sitiotresrios
37 cevada sitioalvorada
42 brocolis fazendapedrabranca
56 ervadoce sitiodospinheiros
54 abacaxi sitiosantaclara
47 gergelim fazendatrescachoeiras
124 capimtifton sitiovaledosol
69 coentro fazendahorizonteazul
176 alface sitiovaledasflores
155 maca fazendavaleencantado
165 limao fazendariolimpo
22 uva sitioparaiso
101 manga fazendacamposereno
11 algodao sitiodorio
170 couveflor fazendadosol
189 banana fazendavaledosol
158 camomila sitiobosqueencantado
82 limao sitiorecantodosol
158 morango sitiodopinhal
46 repolho sitiohorizonte
163 morango fazendabelanatureza
23 beterraba fazendasantoantonio
191 trigo sitioverdejante
97 melao fazendavilaalegre
142 repolho sitioribeirao
137 banana sitiocampolimpo
123 algodao sitiorecantoverde
65 mandioca fazendatrescoracoes
52 lentilha sitioprimaveraverde
28 morango fazendadalagoa
175 canadeacucar fazendanovaesperan√ßa
36 morango fazendavaledossonhos
100 alecrim fazendarecantoalegre
195 maca fazendaserraclara
88 mamona sitiomonteverde
74 inhame sitionovohorizonte
146 tomate fazendabelacolina
120 brocolis fazendadoslagos
120 amendoim fazendaserranegra
2 maracuja fazendadoislagos
180 maca sitioriobonito
118 melao sitiocampoalegre
0
97 137