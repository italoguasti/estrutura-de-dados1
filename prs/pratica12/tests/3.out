InOrder:
97 ( [melao|fazendavilaalegre] )
100 ( [alecrim|fazendarecantoalegre] )
101 ( [manga|fazendacamposereno] )
118 ( [melao|sitiocampoalegre] )
120 ( [brocolis|fazendadoslagos] [amendoim|fazendaserranegra] )
123 ( [algodao|sitiorecantoverde] )
124 ( [capimtifton|sitiovaledosol] )
137 ( [banana|sitiocampolimpo] )

PreOrder:
100 ( [alecrim|fazendarecantoalegre] )
97 ( [melao|fazendavilaalegre] )
118 ( [melao|sitiocampoalegre] )
120 ( [brocolis|fazendadoslagos] [amendoim|fazendaserranegra] )
123 ( [algodao|sitiorecantoverde] )
101 ( [manga|fazendacamposereno] )
137 ( [banana|sitiocampolimpo] )
124 ( [capimtifton|sitiovaledosol] )

PosOrder:
124 ( [capimtifton|sitiovaledosol] )
101 ( [manga|fazendacamposereno] )
97 ( [melao|fazendavilaalegre] )
100 ( [alecrim|fazendarecantoalegre] )
123 ( [algodao|sitiorecantoverde] )
120 ( [brocolis|fazendadoslagos] [amendoim|fazendaserranegra] )
118 ( [melao|sitiocampoalegre] )
137 ( [banana|sitiocampolimpo] )
